cypress/integration/optionalCatchAll_at_root_spec.js

```diff
@@ -104,9 +104,11 @@ describe("pre-rendered pages: /subfolder/[id].js", () => {
   });
 
   it("serves the pre-rendered HTML (and not the Netlify Function)", () => {
-    cy.request("/subfolder/static").then((response) => {
-      expect(response.headers["cache-control"]).to.include("public");
-    });
+    if (Cypress.env("DEPLOY") !== "local") {
+      cy.request("/subfolder/static").then((response) => {
+        expect(response.headers["cache-control"]).to.include("public");
```

Comment:
lowercase cache-control? 🤔

Reply:
Should it be uppercase? 😅 We have it lowercase in `default_spec.js`, too, and it seems to be working:

https://github.com/netlify/next-on-netlify/blob/818cb9196750d8dd06c30157c07b9b169b97064c/cypress/integration/default_spec.js#L653

Reply:
Apparently, headers are supposed to be case-insensitive: https://github.com/cypress-io/cypress/issues/2879#issuecomment-446876062

Cypress transforms all headers to lowercase, so that devs don't have to worry about it: https://github.com/cypress-io/cypress/issues/2879#issuecomment-458862825

Reply:
dope!! merge away!!!!!!